tests/providers/test_internet.py

```diff
@@ -339,6 +340,12 @@ def test_http_method(self, faker, num_samples):
 
         assert expected_methods == sorted(got_methods)
 
+    def test_http_status_code(self, faker, num_samples):
+        provider = InternetProvider(faker)
+        status_code = provider.http_status_code()
+        assert isinstance(status_code, int)
+        assert 100 <= status_code <= 599
+
```

Comment:
can we also have a test for `include_unassigned=False`?

Reply:
OK - done!